### PR TITLE
fix(Switch): stop event propagation on change

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

**Review by sergivillar — APPROVED** [review]

File: src/portal.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const usePortalNodes = (): PortalNodesContext => React.useContext(PortalN
  * This is useful for components that need position fixed, for example
  * for menus, alerts, overlays, etc.
  *
- * @see https://reactjs.org/docs/portals.html
+ * see https://reactjs.org/docs/portals.html
```
Comment (pladaria) [inline src/portal.tsx:50]: unrelated change

**Review by atabel — APPROVED** [review]
